misc/ci_setup.py

```diff
@@ -178,7 +178,6 @@ def run(self):
             "--user",
             "colorama",
             "numpy",
-            "Pillow",
             "scipy",
```

Comment:
As far as I know, scipy is not ready for m1(the build in my mac failed), and there is no usage found in the whole taichi package, so consider remove it?

Reply:
Nice catch! Pls feel free to open a new one to remove this :-)

Reply:
opened in #2373